locales/es-ES.yml

```diff
@@ -0,0 +1,216 @@
+es-ES:
+
+  # global, logged out, and auth-related
+  add: Crear
+  add_new: Crear nuevo
+  already_have_account: Tengo una cuenta # signup form cta
+  back: Atrás
+  buy_now: Comprar
+  cancel: Cancelar
+  change_password: Cambiar Contraseña
+  charge_level: Nivel de Batería
+  charged: Cargada
+  close: Cerrar
+  confirm_change_password: Cambiar mi contraseña
+  confirm_password: Confirmar nueva contraseña
+  create_an_account: Crear cuenta de usuario # login form cta
+  days: días
+  delete: Borrar
+  device_id: Identificador de dispositivo
+  edit: Editar
+  email_address: Correo electrónico
+  first_name: Nombre
+  forgot_password: Olvidé mi contraseña # login form link
+  forgot_password_title: Otra vez # heading on /password/new (hint: https://www.google.com/search?q=here+we+go+again+meme)
+  get_started: ¡Empecemos!
+  identify: Identificar
+  keep_me_signed_in: Mantenerme conectado
+  knowledge_base: Base de Conocimiento
+  last_name: Apellido
+  login: Iniciar sesión
+  log_out: Cerrar sesión
+  minimum_characters: Caracteres como mínimo # ex: 8 characters minimum
```

Comment:
Some sentences change depending on how the fragment is used. For example:

"8 minimum characters" can translate as "Mínimo 8 caracteres" or "8 caracteres como mínimo". The correct translation would depend on how the string is pieced together.